doc-content/enterprise-only/vehicle-route-planning/backend-properties-ref.adoc

```diff
@@ -0,0 +1,72 @@
+[id='backend-properties-ref_{context}']
+
+= OptaWeb Vehicle Routing back end configuration properties
+You can set the application properties listed in the following table.
+
+[cols="m,d,a,d",options="header"]
+|===
+
+|Property
+|Type
+|Example
+|Description
+
+|app.demo.data-set-dir
+|Relative or absolute path
+|/home/user/{data-dir-name}/dataset
```

Comment:
The `{data-dir-name}` attribute is not defined. It should be defined similar to this:
```
:data-dir-name: .optaweb-vehicle-routing
```

Reply:
@yurloc Is the dot needed?
.optaweb-vehicle-routing

Reply:
Oh I see! This is a variable, not a place holder. OK.

Reply:
Defined in the attributes.adoc file

Reply:
The dot is part of the directory name. It's not mandatory but it's convenient because it makes the directory hidden on a linux system.